lib/rubocop/cop/mixin/hash_shorthand_syntax.rb

```diff
@@ -172,6 +173,11 @@ def hash_with_values_that_cant_be_omitted?(hash_value_type_breakdown)
         hash_value_type_breakdown[:value_needed]&.any?
       end
 
+      def ignore_explicit_ommitable_hash_shorthand_syntax?(hash_value_type_breakdown)
```

Comment:
There's no word "ommittable" - I think you meant to use "omissible". :-) See https://www.merriam-webster.com/dictionary/omissible

Reply:
Thanks, TIL, updated f91da7dca4a74a6d3efa1283624296ffde7a8245